app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     buildToolsVersion '23.0.2'
     defaultConfig {
         applicationId 'fr.neamar.kiss'
-        minSdkVersion 10
+        minSdkVersion 11
```

Comment:
Is there any way to keep Android 2.3.3+ compatibility with these changes?
We discussed this a couple of weeks ago and people came to the conclusion that we want to keep SDK 10 compatibility for now. (It's not a problem if the documents provider is always disabled for this platform, btw.)

@Neamar: Comments on this?